source/objects/AccelStruct.cpp

```diff
@@ -411,7 +412,6 @@ World::World(GarrysMod::Lua::ILuaBase* LUA, const std::string& mapName)
 
 		memcpy(tri.normals, normals + vi0, sizeof(glm::vec3) * 3);
 		memcpy(tri.tangents, tangents + vi0, sizeof(glm::vec3) * 3);
-		memcpy(tri.uvs, uvs + vi0, sizeof(glm::vec2) * 3);
 		memcpy(tri.alphas, alphas + vi0, sizeof(float) * 3);
```

Comment:
Would be ideal to pass the other data in via the constructor too, but not gonna make you do that